CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.3.0] - 2022-05-12
 
 ### Added
-
+- Add support for live transcriptons in GovCloud regions. 
```

Comment:
Two bullet points:
- Add support for hosting meetings in US GovCloud regions
- Add support for starting live transcription in US GovCloud regions

Reply:
updated.